cmd/server/upsClient.go

```diff
@@ -15,8 +15,34 @@ type upsClient struct {
 
 const BaseUrl = "http://localhost:8080/rest/applications"
 
+// Delete the variant with the given google key
+func (client *upsClient) deleteAndroidVariant(key string) bool {
```

Comment:
this is pretty generic, and working for all variant types;  let's just name it `deleteVariant`

Reply:
👍